src/server/services/wfs/qgswfsutils.cpp

```diff
@@ -153,6 +153,35 @@ namespace QgsWfs
       request.setFlags( QgsFeatureRequest::ExactIntersect | QgsFeatureRequest::NoFlags );
       return request;
     }
+    else if ( filterElem.firstChildElement().tagName() == QLatin1String( "And" ) &&
```

Comment:
does capitalization matter?

Reply:
The standard says : 
```
<xsd:element name="And"
type="fes:BinaryLogicOpType"
substitutionGroup="fes:logicOps"/>
```
As far as "BBOX" is tested with case sensitive matching, I've done the same for "And".

Reply:
Note that in QgsOgcUtils, "And" is also tested with case sensitive matching.
https://github.com/qgis/QGIS/blob/9cf6bbf42490da69d7513f5de2c6e45e9a8ab652/src/core/qgsogcutils.cpp#L1654:L1673